staging/src/k8s.io/controller-manager/service/config/BUILD

```diff
@@ -7,7 +7,8 @@ go_library(
         "types.go",
         "zz_generated.deepcopy.go",
     ],
-    importpath = "k8s.io/kubernetes/pkg/controller/service/config",
+    importmap = "k8s.io/kubernetes/vendor/k8s.io/controller-manager/service/config",
```

Comment:
service/config used to have things like an OWNERS file. Is there a reason we seem to have lost that?

Reply:
Thanks for the catch. Updated^